alicloud/data_source_alicloud_db_instance_classes.go

```diff
@@ -93,6 +97,10 @@ func dataSourceAlicloudDBInstanceClasses() *schema.Resource {
 							Type:     schema.TypeString,
 							Computed: true,
 						},
+						"price": {
```

Comment:
please add a parameter `sorted_by`. If the value is "price", use price to sort them, otherwise, don't sort.